styles.css

```diff
@@ -2,6 +2,250 @@
     box-sizing: border-box;
 }
 
-body {
-    background-color: rgb(234, 234, 234);
+body{
+    /* background-color: black; */
+    color: aliceblue;
```

Comment:
this color should not be applied on the body. Rather it should be applied more specifically on the banner section only.

Reply:
yes i changed it to the banner section alone